src/Admin/Content/StaticContentAdmin.php

```diff
@@ -11,22 +11,40 @@
 
 namespace Symfony\Cmf\Bundle\SonataPhpcrAdminIntegrationBundle\Admin\Content;
 
+use Ivory\CKEditorBundle\Form\Type\CKEditorType;
 use Sonata\AdminBundle\Datagrid\DatagridMapper;
 use Sonata\AdminBundle\Datagrid\ListMapper;
 use Sonata\AdminBundle\Form\FormMapper;
 use Symfony\Cmf\Bundle\ContentBundle\Model\StaticContentBase;
 use Symfony\Cmf\Bundle\SonataPhpcrAdminIntegrationBundle\Admin\AbstractAdmin;
 use Symfony\Cmf\Bundle\TreeBrowserBundle\Form\Type\TreeSelectType;
-use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
 
 class StaticContentAdmin extends AbstractAdmin
 {
     protected $translationDomain = 'CmfSonataPhpcrAdminIntegrationBundle';
 
+    /**
+     * @var []
+     */
+    private $ckEditorConfig;
+
     public function getExportFormats()
     {
-        return array();
+        return [];
+    }
+
+    /**
+     * Configuration passed to the CKEditorType. Have a look at the docs:
+     *
+     * http://symfony.com/doc/master/bundles/IvoryCKEditorBundle/usage/config.html
+     *
+     * @param [] $config
```

Comment:
please fix the typehint here and explain what $config is

what are the options for the ckeditor type? does it have other options than config_name?